timemachine/cpp/src/nonbonded_all_pairs.hpp

```diff
@@ -29,44 +31,44 @@ typedef void (*k_nonbonded_fn)(
 template <typename RealType, bool Interpolated> class NonbondedAllPairs : public Potential {
 
 private:
-    std::array<k_nonbonded_fn, 16> kernel_ptrs_;
+    const int N_;
```

Comment:
Would be good to have a comment here to distinguish it from `K_`

Reply:
Good call, added in db146c6